webroot/src/components/CollapseCaretButton/CollapseCaretButton.less

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
_UI testing find - just commenting here for a resolvable comment_

I noticed that clicking the actual checkbox on the privilege-select screen doesn't appear to bubble up to cause the item to be selected. If this is an easy fix, you can address it here. If not, you can open a new ticket in the backlog that the team can prioritize in tomorrow's refinement.

<img width="293" alt="Screenshot 2024-11-11 at 10 27 18 AM" src="https://github.com/user-attachments/assets/3e2015ca-e304-4003-9ba7-25e11ebc53c7">

Reply:
I have noticed this as well, I was planning on fixing this while doing the ticket to alter enhance the mobile styles

Reply:
Added to https://github.com/csg-org/CompactConnect/issues/297